cScripts/CavFnc/functions/init/fn_init_vehicle.sqf

```diff
@@ -0,0 +1,38 @@
+#include "..\script_component.hpp";
+/*
+ * Author: CPL.Brostrom.A
+ * This function add eventhandelers adding cav functionality to all vehicles.
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * call cScripts_fnc_init_vehicle
+ *
+ * Public: No
+ */
+
+#ifdef DEBUG_MODE
+    ["Applying Event Handers (init) to vehicles for function expantions...", "InitVehicle"] call FUNC(info);
+#endif
+
+["AllVehicles", "init", {
+    _this params ["_vehicle"];
+    if (_vehicle iskindOf "man") exitWith {};
+    _vehicle call EFUNC(vehicle,addFunctions);
+    _vehicle call EFUNC(vehicle,addInventory);
+    _vehicle call EFUNC(vehicle,addCosmetics);
+    _vehicle call EFUNC(vehicle,addRadio);
+    _vehicle call EFUNC(vehicle,addStagingActions);
+}, true, [], true] call CBA_fnc_addClassEventHandler;
```

Comment:
Improve this functions the load is some what slow now. Possible run add Radio Separatly or spawn it.